### ISSUE TITLE
Tests for Remove-TssFolderTemplate don't recognize  parameters Id and TemplateId

### ISSUE THREAD

**Issue by peetrike** [thycotic-ps/thycotic.secretserver#80]
**Describe the bug**
Tests for Remove-TssFolderTemplate don't recognize parameters Id and TemplateId.

**To Reproduce**
1. PowerShell script:

```powershell
Invoke-Pester -Path .\tests\folders\Remove-TssFolderTemplate.Tests.ps1 -Output Detailed
```

1. Error/issue observed:

```powershell
Starting discovery in 1 files.
Discovering in C:\Projects\GitHub\thycotic.secretserver\tests\folders\Remove-TssFolderTemplate.Tests.ps1.
Found 4 tests. 81ms
Discovery finished in 86ms.

Running tests from 'C:\Projects\GitHub\thycotic.secretserver\tests\folders\Remove-TssFolderTemplate.Tests.ps1'
Describing Remove-TssFolderTemplate verify parameters
 Context Verify parameters
   [+] Remove-TssFolderTemplate should contain TssSession parameter 19ms (15ms|4ms)
   [-] Remove-TssFolderTemplate should not contain parameter: Id 29ms (29ms|1ms)
    Expected $null or empty, but got Id.
```

**Expected behavior**
Tests should accept all parameters present in function

**Possible Fix**

Adding missing parameters to line:
https://github.com/thycotic-ps/thycotic.secretserver/blob/23b93afacf47312c8c3d2f20104a617f790f4bc7/tests/folders/Remove-TssFolderTemplate.Tests.ps1#L7
fixed the problem.

**Environment (please complete the following information):**

```powershell
Name                           Value
----                           -----
PSVersion                      7.1.2
PSEdition                      Core
GitCommitId                    7.1.2
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

***Host information:***

[ ] Powershell.exe
[x] pwsh.exe
[ ] PowerShell ISE
[ ] VS Code (Insiders/Stable)

**Additional context**

**Comment by wsmelton** [comment]
Resolved, and will be pushed up this afternoon.

**Comment by wsmelton** [comment]
Fixed in  7eeb8d8 

https://github.com/thycotic-ps/thycotic.secretserver/blob/7eeb8d87783833502c1e47650f66e611df71aae1/tests/folders/Remove-TssFolderTemplate.Tests.ps1#L7
